test/e2e/basic.test.ts

```diff
@@ -0,0 +1,22 @@
+import { resolve } from "pathe";
```

Comment:
I guess additional tests are unnecessary it seems mostly a specific bug as I tend to keep tests simple for now.

Reply:
:+1: i'll revert it